src/api/routes/habit.routes.js

```
@@ -1,21 +1,29 @@
-// Description: API를 위한 라우터 설정 코드 파일입니다.
+// Description: 습관(Habit) 관련 API 라우터 설정 파일입니다.
+
+// 라이브러리 정의
 import express from 'express';
-import coreMiddleware from '../../../src/common/cors.js';
-import errorMiddleware from '../../../src/common/error.js'; // 에러를 추가할 일이 있다면, 해당 파일에 케이스를 추가해주시기 바랍니다.
+
+// 미들웨어 정의
+import corsMiddleware from '../../common/cors.js';
```

ESLint

[error] Unexpected use of file extension "js" for "../../common/cors.js"

(import/extensions)